### PR TITLE
protocol(ticdc): remove useless code and setup DefaultMaxMessageBytes

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve?

Issue Number: None

### What is changed and how it works?

remove useless code.

### Check List 

Tests 

 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change

Side effects

None

Related changes

None

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by 3AceShowHand — APPROVED** [review]

File: cdc/sink/producer/kafka/kafka.go [hunk 1]
```diff
@@ -61,8 +61,8 @@ type Config struct {
 func NewConfig() *Config {
 	return &Config{
 		Version: "2.4.0",
-		// MaxMessageBytes will be used to initialize producer, we set the default value (1M) identical to kafka broker.
-		MaxMessageBytes:   1 * 1024 * 1024,
+		// MaxMessageBytes will be used to initialize producer.
+		MaxMessageBytes:   config.DefaultMaxMessageBytes,
```
Comment (amyangfei) [inline cdc/sink/producer/kafka/kafka.go:65]: Is there unit test to cover this logic?

**Review by overvenus — APPROVED** [review]
LGTM

**Pull request merged**